test/unit/foreman/renderer/scope/report_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class ReportScopeTest < ActiveSupport::TestCase
+  setup do
+    source = Foreman::Renderer::Source::String.new(content: '')
+    @scope = Foreman::Renderer::Scope::Report.new(source: source)
+  end
+
+  describe '#report_render' do
+    test 'render simple values' do
+      @scope.report_row('Col1': 'Val1', 'Col2': 'Val2')
```

Comment:
Could you please add test where user specifies different headers, verifying that first row defines headers?